src/network/netdev/bridge.c

```diff
@@ -230,6 +236,40 @@ int config_parse_bridge_port_priority(
                         prio);
 }
 
+int config_parse_bridge_fdb_max_learned(
+                const char *unit,
+                const char *filename,
+                unsigned line,
+                const char *section,
+                unsigned section_line,
+                const char *lvalue,
+                int ltype,
+                const char *rvalue,
+                void *data,
+                void *userdata) {
+        Bridge *b = ASSERT_PTR(userdata);
+        int r;
+
+        assert(filename);
+        assert(lvalue);
+        assert(rvalue);
+        assert(data);
+
+        b->fdb_max_learned_set = false;
+
+        if (isempty(rvalue))
+                return 0;
+
+        r = config_parse_uint32_bounded(unit, filename, line, section, section_line, lvalue, rvalue,
+                                        0, UINT32_MAX, true, &b->fdb_max_learned);
+
```

Comment:
Please drop the empty line. We usually write error handling soon after function call.